packages/vuetify/src/components/VSlideGroup/VSlideGroup.ts

```diff
@@ -382,18 +382,20 @@ export const BaseSlideGroup = mixins<options &
       }, this.$vuetify.rtl, this.scrollOffset)
     },
     setWidths /* istanbul ignore next */  () {
-      window.requestAnimationFrame(() => {
-        const { content, wrapper } = this.$refs
-
-        this.widths = {
-          content: content ? content.clientWidth : 0,
-          wrapper: wrapper ? wrapper.clientWidth : 0,
-        }
-
-        this.isOverflowing = this.widths.wrapper < this.widths.content
-
-        this.scrollIntoView()
-      })
+      if (typeof window === 'undefined') {
```

Comment:
Avoid calling global window, instead use a pattern calling the `$el.ownerDocument.defaultView` which is the current `WindowProxy` (or `Window` global in its frame context).

Also, avoid re-defining object properties, in Vue 2 we have `Vue.set` and `this.$set` methods in case.

```patch
diff --git a/packages/vuetify/src/components/VSlideGroup/VSlideGroup.ts b/packages/vuetify/src/components/VSlideGroup/VSlideGroup.ts
index 1a6693be7..c81c0270e 100644
--- a/packages/vuetify/src/components/VSlideGroup/VSlideGroup.ts
+++ b/packages/vuetify/src/components/VSlideGroup/VSlideGroup.ts
@@ -385,5 +385,7 @@ export const BaseSlideGroup = mixins<options &
-      if (typeof window === 'undefined') {
-        this.widths = { content: 0, wrapper: 0 }
-        setTimeout(() => this.setWidths(), 1000)
-      } else if (this.widths === { content: 0, wrapper: 0 }) {
-        window.requestAnimationFrame(() => {
+      // defaultView represent window. But this code is sensitive to
+      // only running when it is client-side, but using Vue.js' so
+      // we do not rely on global object, and will make this code
+      // ALSO work when SSRing
+      const defaultView: Window | null = this.$el && this.$el.ownerDocument ? this.$el.ownerDocument.defaultView : null
+      if (defaultView) {
+        defaultView.requestAnimationFrame(() => {
@@ -391,4 +393,6 @@ export const BaseSlideGroup = mixins<options &
-          this.widths = {
-            content: typeof content !== 'undefined' ? content.clientWidth : 0,
-            wrapper: typeof wrapper !== 'undefined' ? wrapper.clientWidth : 0,
-          }
+          // Avoid re-defining existing objects, this'll override reactive attributes
+          this.widths.content = content && content.clientWidth ? content.clientWidth : 0
+          this.widths.wrapper = wrapper && wrapper.clientWidth ? wrapper.clientWidth : 0
+          // Otherwise, use Vue.set or this.$set
+          // this.$set(this.widths, 'content', typeof content !== 'undefined' ? content.clientWidth : 0)
+          // this.$set(this.widths, 'wrapper', typeof wrapper !== 'undefined' ? wrapper.clientWidth : 0)
@@ -397,0 +402,4 @@ export const BaseSlideGroup = mixins<options &
+      } else {
+        this.widths.content = 0
+        this.widths.wrapper = 0
+        setTimeout(() => this.setWidths(), 1000)
```

Reply:
That is why I modified the source, in which the `window` (not "`Window`") object was called without prior assignment, allocation, reference etc.

Reply:
Just reflected upon.
Much appreciated your insight

Reply:
When I said capitalized Window, i was making reference to the type.